Big-Spender/readme.md

```diff
@@ -93,6 +112,14 @@ CREATE YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
+
+
+                SELECT SUM(spends.amount) AS total_amount
+                FROM spends
+                INNER JOIN suppliers ON suppliers.id = spends.supplier_id
```

Comment:
Great job with the joins! However, two columns are expected in the results. What additional information from the suppliers table should be included in the SELECT statement?